<launch>

  <arg name="display_camera_window" default="false"/>

  <node pkg="homework1" type="face_detector.py" name="face_detector" output="screen">
    <param name="display_camera_window" type="bool" value="$(arg display_camera_window)" />
    <!-- The path to the haarcascade_face.xml used in HaarDetector -->
    <param name="haar_cascade_data_file_path" type="string" value="$(find exercise4)/scripts/detectors/data/haarcascade_face.xml" />
  </node>

</launch>
